src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -867,7 +867,6 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
   render() {
     // languate=HTML
     return html`
-      <lablup-loading-spinner id="loading-spinner"></lablup-loading-spinner>
```

Comment:
Any reason to remove `lablup-loading-spinner` from here ?

Reply:
This is because it wasn't used.